libs/hmpb/src/ballot_templates/vx_default_ballot_template.tsx

```diff
@@ -323,11 +318,7 @@ function CandidateContest({
                   borderTop: `1px solid ${Colors.DARK_GRAY}`,
                 }}
               >
-                {/* Match line-height of text to align bubble to center of write-in candidate name */}
-                <BubbleWrapper
-                  optionInfo={optionInfo}
-                  style={{ height: '1.25rem' }}
```

Comment:
Note that the 1.25rem here changes to the default 1.2rem in `AlignedBubble`. This is the main change that causes fixture and snapshot updates in the rest of this PR. Visually it's basically impercetible